app/components/agenda/agendaitem/agendaitem-postponed.js

```diff
@@ -0,0 +1,97 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import addWeeks from 'date-fns/addWeeks';
+import { PAGE_SIZE } from 'frontend-kaleidos/config/config';
+
+/**
+ * @argument subcase
+ * @argument meeting
+ * @argument agendaActivity
+ */
+export default class AgendaitemPostponed extends Component {
+  @service store;
+  @service agendaService;
+
+  @tracked canPropose = false;
```

Comment:
Can't `canPropose` be derived from `modelsForProposedAgenda` being undefined? Feels a bit like we're maintaining duplicate state here or am I missing something?

Reply:
sounds good